cardinal/config.go

```diff
@@ -0,0 +1,28 @@
+package cardinal
+
+import "os"
+
+type WorldConfig struct {
+	RedisAddress       string
+	RedisPassword      string
+	CardinalWorldId    string
```

Comment:
Should this just be `CardinalNamespace` and have the optional namespace be removed?

Reply:
I agree. I can do that